### PR TITLE
Add compat data for Response (Fetch API)

### PR DESCRIPTION
**Pull request by maboa**
[https://developer.mozilla.org/en-US/docs/Web/API/Response](https://developer.mozilla.org/en-US/docs/Web/API/Response)

### CODE REVIEW

File: api/Response.json [hunk 1]
```diff
@@ -0,0 +1,1131 @@
...
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/Response.json]: `"opera_android"` should match `"opera"`.

Comment (maboa) [reply api/Response.json]: 👍

File: api/Response.json [hunk 2]
```diff
@@ -0,0 +1,1142 @@
...
+            "version_added": "10.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "body_formdata": {
```
Comment (Elchi3) [inline api/Response.json]: Let's remove this. It is stored here https://github.com/mdn/browser-compat-data/blob/master/api/Body.json#L380

File: api/Response.json [hunk 3]
```diff
@@ -0,0 +1,1142 @@
...
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          }
+        }
+      },
+      "redirected_attribute": {
```
Comment (Elchi3) [inline api/Response.json]: Remove this, it is redundant with the entry for `redirected` below.

File: api/Response.json [hunk 4]
```diff
@@ -0,0 +1,1142 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 5]
```diff
@@ -0,0 +1,1142 @@
+{
+  "api": {
+    "Response": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Response",
+        "support": {
+          "webview_android": {
```
Comment (Elchi3) [inline api/Response.json]: I think this should be

```json
          "webview_android": [
            {
              "version_added": "42"
            },
            {
              "version_added": "41",
              "flags": [
                {
                  "type": "preference",
                  "name": "Experimental Web Platform Features"
                }
              ]
            }
          ],
          "chrome": [
            {
              "version_added": "42"
            },
            {
              "version_added": "41",
              "flags": [
                {
                  "type": "preference",
                  "name": "Experimental Web Platform Features"
                }
              ]
            }
          ],
          "chrome_android": [
            {
              "version_added": "42"
            },
            {
              "version_added": "41",
              "flags": [
                {
                  "type": "preference",
                  "name": "Experimental Web Platform Features"
                }
              ]
            }
],
```

File: api/Response.json [hunk 6]
```diff
@@ -0,0 +1,1142 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 7]
```diff
@@ -0,0 +1,1142 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 8]
```diff
@@ -0,0 +1,1142 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 9]
```diff
@@ -0,0 +1,1142 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 10]
```diff
@@ -0,0 +1,1142 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 11]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 12]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 13]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 14]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 15]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 16]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 17]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 18]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 19]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 20]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 21]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 22]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 23]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 24]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 25]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 26]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 27]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 28]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 29]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 30]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 31]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox_android": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 32]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 33]
```diff
@@ -0,0 +1,1142 @@
...
+                "chrome": [
+                  {
+                    "version_added": "42"
+                  },
+                  {
+                    "version_added": "41",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json [hunk 34]
```diff
@@ -0,0 +1,1142 @@
...
+                "firefox": [
+                  {
+                    "version_added": "39"
+                  },
+                  {
+                    "version_added": "34",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: dom.fetch.enabled

File: api/Response.json [hunk 35]
```diff
@@ -0,0 +1,1142 @@
...
+                "opera": [
+                  {
+                    "version_added": "29"
+                  },
+                  {
+                    "version_added": "28",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": ""
```
Comment (Elchi3) [inline api/Response.json]: "Experimental Web Platform Features"

File: api/Response.json (hunk 5 above)
Comment (maboa) [reply api/Response.json]: 👍

File: api/Response.json (hunk 2 above)
Comment (maboa) [reply api/Response.json]: 👍

File: api/Response.json (hunk 3 above)
Comment (maboa) [reply api/Response.json]: 👍

**Comment by maboa** [comment]
Should be good to go now! :)

**Pull request merged**